deploy/resources/devel/optional/pipelinesamples/docker_build_deploy.yaml

```diff
@@ -14,30 +14,43 @@ spec:
     apiVersion: tekton.dev/v1beta1
     kind: Pipeline
     metadata:
-      name: docker-build-and-deploy
+      name: docker-build-and-deploy-pr
     spec:
       params:
         - name: IMAGE_NAME
           type: string
-      resources:
-        - name: app-source
-          type: git
-        - name: app-image
-          type: image
-
+        - name: GIT_REPO
```

Comment:
Same question as in https://github.com/openshift/tektoncd-pipeline-operator/pull/386#pullrequestreview-460780261. Shouldn't we also allow the user to pass a revision ?

Reply:
PR updated according to comments